chcemvediet/apps/anonymization/management/commands/attachment_anonymization.py

```diff
@@ -0,0 +1,65 @@
+from optparse import make_option
+
+import magic
+from django.core.files.base import ContentFile
+from django.core.management.base import BaseCommand, CommandError
+
+from poleno.attachments.models import Attachment
+from chcemvediet.apps.anonymization.models import AttachmentFinalization
+from chcemvediet.apps.inforequests.models import Action
+
+
+class Command(BaseCommand):
+    help = u'Creates AttachmentFinalization for the specified Attachment.'
```

Comment:
@viliambalaz Vid zadanie:
> v dokumentacii commandu vsade pis o anonymizacii, nie o finalizacii. Kedze z pohladu externych skriptov je to cele anonymizacia. Slovo finalizacia je iba nasa technikalita v ramci anonymizacnej app.

Naozaj nasho admina, co bude vytvarat anonymizacie priloh, nezaujima ako sa technicky vola model, do ktoreho tie data ukladame. Jeho zaujima semantika toho modelu, teda ze to je anonymizacia prilohy.